recipe/meta.yaml

```diff
@@ -38,129 +42,142 @@ build:
+    - {{native}}pango                    # [not win]
+    - {{native}}icu
+    - {{native}}pcre
+    - {{native}}tk
+    - {{native}}zlib 1.2.8               # [win]
+    - {{native}}zlib                     # [not win]
+    - {{native}}gsl
+    - {{native}}libtiff >=4.0.3,<4.0.8   # [win]
+    - {{native}}libtiff                  # [not win]
+    - {{native}}libxml2                  # [win]
+    - {{native}}libxml2                  # [not win]
+    - {{native}}libssh2                  # [not win]
+    - {{native}}krb5                     # [not win]
+    - {{native}}openblas-devel >=0.2.20  # [linux or osx]
+    - {{native}}openblas                 # [win]
 
   run:
-    - readline                 # [not win]
-    - ncurses                  # [not win]
-    - jpeg                     # [not win]
-    - curl                     # [not win]
-    - xz                       # [not win]
-    - libpng                   # [not win]
-    - cairo                    # [not win]
-    - libssh2                  # [not win]
-    - openssl                  # [not win]
-    - krb5                     # [not win]
-    - {{native}}bzip2 1.0.*    # [not osx]
-    - {{native}}libjpeg-turbo  # [win]
-    - {{native}}libiconv       # [win]
-    - {{native}}gmp 6.1.*      # [win]
-    - {{native}}fftw           # [win]
-    - {{native}}xz             # [win]
-    - {{native}}mpfr 3.1.*     # [win]
-    - {{native}}libsndfile     # [win]
-    - {{native}}bwidget        # [win]
-    - {{native}}tktable        # [win]
-    - {{native}}libpng         # [win]
-    - pango                    # [not win]
-    - icu                      # [not win]
-    - pcre                     # [not win]
-    - {{native}}pcre           # [win]
-    - {{native}}tk             # [win]
-    - tk                       # [not win]
-    - zlib                     # [not win]
-    - {{native}}zlib           # [win]
-    - {{native}}gsl            # [win]
-    - gsl                      # [not win]
-    - libtiff                  # [not win]
-    - {{native}}libtiff        # [win]
-    - {{native}}libxml2 2.9.*  # [win]
-    - libxml2                  # [not win]
-    - xorg-libx11              # [linux]
-    - xorg-libxt               # [linux]
-    - xorg-libxrender          # [linux]
-    - xorg-libxext             # [linux]
-    - xorg-libxdmcp            # [linux]
+    - _r-mutex 1.* anacondar_1
+    # Needed by R CMD javareconf
+    - make                               # [not win]
+    - {{ compiler('fortran') }}          # [not win]
+    - {{ compiler('cxx') }}              # [not win]
+    - {{ compiler('c') }}                # [not win]
+    - llvm-openmp >=4.0.1                # [osx]
+    - {{native}}gcc-libs                 # [win]
+    - curl                               # [not win]
+    - {{native}}bzip2 1.0.*              # [win]
+    - {{native}}libjpeg-turbo            # [win]
+    - {{native}}libiconv                 # [win]
+    - {{native}}libuuid                  # [linux]
```

Comment:
Can you remove `{{native}}` here? conda-build doesn't recognize that `libuuid` is a used variable with `{{native}}` in front.

Reply:
Then that is what needs to be fixed. Please consider a PR to conda-build to fix the real problem.

Why does it matter here though here since `libuuid` is not part of a build matrix?